Gemfile

```diff
@@ -59,7 +59,9 @@ gem 'sparql',         '~> 1.99'
 gem 'json-ld',        '~> 1.99'
 
 group :development, :test do
-  gem 'rdf-do',       '~> 1.99' # Using DataObjects for SPARQL store
-  gem 'data_objects', '~> 0.10'
-  gem 'do_sqlite3' ,  '~> 0.10'
+  #gem 'rdf-do',       '~> 1.99' # Using DataObjects for SPARQL store
+  #gem 'data_objects', '~> 0.10'
+  #gem 'do_sqlite3' ,  '~> 0.10'
+  gem 'rdf-mongo'
+  gem 'bson_ext'
 end
```

Comment:
Thanks for clarification and that makes sense.  I'd be interested in some general enlightenment about the architecture considerations - curious about this approach and generally how an RDF/SPARQL approach can provide additional services that Solr can't provide.  I'm assuming this tackles problems that are not related to just pushing triple store data into Solr (or a third party sync, like FC4 via messaging).